### ISSUE TITLE
Build system misconfiguration for refdesign with Coyote + RDMA + U250

### ISSUE THREAD

**Issue by Tobi-Alonso** [Xilinx/ACCL#183]
# Reprodution
```sh
source <VITIS_INSTALL>/settings64.sh
cd "test/refdesigns"
make MODE=coyote_rdma PLATFORM=xilinx_u250_gen3x16_xdma_4_1_202210_1
```

# Error message
"Number of DDR channels misconfigured"

# Details
N_DDR_CHAN is not correctly set for U250 + RDMA in 'test/refdesigns/Coyote/hw/config.cmake'

**Comment by quetric** [comment]
Thanks @Tobi-Alonso in fact N_DDR_CHAN is not set correctly for any non-HBM platform

**Comment by quetric** [comment]
fixed in 343da3ef9ab86851
